CHANGELOG.md

```diff
@@ -568,10 +560,7 @@ NOTE: This release has a number of fixes compared to v10.3.0-rc.1. For a complet
 * File format: generates Realms with format v20 (reads and upgrades file format v5 or later for non-synced Realm, upgrades file format v10 for synced Realms).
 
 ### Internal
-* Upgraded Realm Core from v10.7.1 to 10.7.1.
-* Upgraded Realm Core from v10.6.0 to 10.7.1
-* Upgraded Realm Core from v10.1.3 to v10.1.4.
-* Upgraded Realm Sync from v10.1.4 to v10.1.5.
```

Comment:
I think this line should stay.  Otherwise we're missing the upgrade from 10.1.4 to 10.1.5

Reply:
You are right: the upgrade was done in https://github.com/realm/realm-js/commit/484ac95f51f4a9bbda6f238fb215f0ace5b70e56